includes/controller/users_controller.php

```diff
@@ -79,6 +79,13 @@ function user_delete_controller()
         }
 
         if ($valid) {
+            // Move user created news/answers/worklogs/shifts  etc. to deleting user
+            $user_source->news()->update(['user_id' => $user->id]);
+            $user_source->questionsAnswered()->update(['answerer_id' => $user->id]);
+            $user_source->worklogsCreated()->update(['creator_id' => $user->id]);
+            $user_source->shiftsCreated()->update(['created_by' => $user->id]);
+            $user_source->shiftsUpdated()->update(['updated_by' => $user->id]);
+
```

Comment:
logs could be reassigned too
confirmed_id of user angel types is missing

Reply:
Having the logs there too would, imho, move too much "other users content" over so i don't think that this is a good idea